pcdsdevices/lodcm.py

```diff
@@ -60,6 +65,74 @@ def __init__(self, prefix, *args, **kwargs):
         super().__init__(prefix, *args, **kwargs)
 
 
+class CHI1(InOutRecordPositioner):
```

Comment:
These guys here all have only `Unknown`, `C` and `Si` states - are they still considered `InOUtRecordPositioners` or should they be something else? Also, does this mean that they don't move in and out of the beamline? what's their role?

Reply:
It does seems surprising there are no "out" states for these... Someone who's used them should be able to chime in to clarify.

Reply:
chi is a rotation axis, so having no "out" state makes sense.

I'm not 100% sure what they are rotating (surely it's the crystal) or why there are states at all if it is the crystal. Maybe its a nominally aligned position at a known energy?

Reply:
@cristinasewell, would you mind adding a class docstring indicating the reason for the lack of the out state?